### ISSUE TITLE
Source Maps Not Working with PostCSS

### ISSUE THREAD

**Issue by oompt** [oompt/sage-lost-stylus#1]
I turned off source maps (for CSS), as it was not playing nicely with PostCSS. 

I don't use source maps much for CSS, but I'm still interested in figuring out a solution for this.

**Comment by hybridvision** [comment]
Hello,

Thanks very much for this fork - Sage is great but with Stylus and Lost, it's even better :)

I am curious to know what the status of this issue is - your updates in December seem to make the sourcemaps generate but no matter what I do, they won't appear in the browser for me - it's as if the sourcemap is never linked correctly. I see no mention of it in the generated CSS so maybe that's the issue. I don't know enough about how it all fits together though.

As a side note, there was an error in the gulpfile.js because in your last commit you removed the line `var csswring     = require('csswring');` but it still calls `csswring()`. I also noticed in package.json, you have csswring listed twice with different versions...

**Comment by hew** [comment]
Yeah it's kind of a mess inside that Gulp file as stands now. I don't have the time currently, but chances are for my next Wordpress project I will figure this out and turn this project into a more complete/coherent solution. 

So, if you want, stay tuned 😎

**Comment by hybridvision** [comment]
Great, thanks - I also don't have much time at the moment but I made it work well enough for my purposes (I also added the Rupture library for handling breakpoints).

I'll come back to it when my project is finished and see if I can figure out some fixes. I'll also stay tuned :) Cheers!
